.github/workflows/release.yml

```diff
@@ -0,0 +1,172 @@
+name: release
+on:
+  workflow_call:
+    inputs:
+      tag:
+        description: 'image tag prefix'
+        default: 'rc'
+        required: true
+        type: string
+      controller:
+        description: 'controller name'
+        required: true
+        type: string
+    secrets:
+      GHCR_TOKEN:
+        description: 'GitHub Container Registry token'
+        required: true
+      DOCKER_FLUXCD_PASSWORD:
+        description: 'Docker Hub password'
+        required: true
+      GITHUB_TOKEN:
+        description: 'GitHub token for releases'
+        required: true
```

Comment:
Please rename this to match what's in main: `github-token`, `dockerhub-token`, `ghcr-token`. I also don't think we need the GHCR static credential here.